pandas/tests/frame/methods/test_replace.py

```diff
@@ -1610,14 +1597,6 @@ def test_replace_dict_category_type(self, input_category_df, expected_category_d
 
         tm.assert_frame_equal(result, expected)
 
-    def test_replace_with_compiled_regex(self):
```

Comment:
this is unrelated to the current PR, right?  if you want to make a separate PR to parametrize this test, go for it.  better to just use frame_or_series in this file and remove the version in the series tests

Reply:
Sorry, yeah its unrelated. Undone, and just used frame_or_series here, just trying to organize it properly (evidently was misguided :D )

Reply:
> evidently was misguided

not at all, organizing/parametrizing/breaking-large-ancient-tests-into-specific-tests is something we very much want to encourage.  just in dedicated PRs

Reply:
I see, definitely makes sense - thanks